apps/backoffice-v2/src/pages/Entity/components/DirectorsCallToAction/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyObject } from '@ballerine/ui';
+
+export const getRevisionReasonsForDocument = (
+  { type, category }: AnyObject,
+  workflow: AnyObject,
+) => {
+  if (category === 'proof_of_identity' && type === 'passport')
+    return [
+      'Blurry image',
+      'Bad quality photo',
+      'Wrong document',
+      'Copy of a copy',
+      'Cut document',
+    ];
+
+  if (category === 'proof_of_identity_ownership' && type === 'selfie') {
+    return [
+      'Blurry image',
+      'Bad quality photo',
+      'Wrong document',
+      'Copy of a copy',
+      'Person in the selfie does not match ID',
+      'There was no person in the photo',
+    ];
+  }
```

Comment:
This should be handled by schemas or the workflows service and not the backoffice.

Reply:
This is sounds like additional feature to me. In order to do so we have to update schemas and add reasons to schemas in common package. Consider open task for it.